test/remote/gateways/remote_realex_test.rb

```diff
@@ -75,7 +75,7 @@ def test_realex_purchase_with_invalid_login
   end
 
   def test_realex_purchase_with_invalid_account
-    response = RealexGateway.new(fixtures(:realex_with_account)).purchase(@amount, @visa,
+    response = RealexGateway.new(fixtures(:realex_with_account).merge(account: 'invalid')).purchase(@amount, @visa,
       :order_id => generate_unique_id,
       :description => 'Test Realex purchase with invalid acocunt'
```

Comment:
invalid acocunt 😆

Reply:
Ahh, good catch! Apparently that's been in the code for 12 years now. 🤣